docs/releasing.md

```diff
@@ -48,14 +48,26 @@ For version v0.x.y:
 
 #### Automated Process
 
-1. Create an annotated tag `git tag -a -s v0.x.y -m v0.x.y`.
-1. Push the tag to the GitHub repository `git push origin v0.x.y`
+1. Create annotated tags:
```

Comment:
can you please update ####Manual Process notes similarly as well?

Reply:
Done 👍